app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -121,53 +99,43 @@ fun DetailHeader(cardFace: CardFace, onBackClick: () -> Unit) {
 fun DetailFlipCard(
     modifier: Modifier = Modifier,
     cardFace: CardFace,
-    onClickToList: () -> Unit,
     item: TopicItem,
+    onClickToList: () -> Unit,
     onClickBookmark: (Boolean) -> Unit
 ) {
     var scale by remember { mutableStateOf(1f) }
     var rotation by remember { mutableStateOf(1f) }
 
-    FlipCard(
-        rotation = rotation,
+    FlipCard(rotation = rotation,
         modifier = modifier
             .width(dimensionResource(id = R.dimen.selected_card_width))
             .aspectRatio(0.7f)
             .graphicsLayer(
-                scaleX = scale,
-                scaleY = scale,
-                cameraDistance = 12f
+                scaleX = scale, scaleY = scale, cameraDistance = 12f
             )
-            .background(CategoryLevelDummy.valueOf(level).backgroundColor),
+            .background(TopicLevel.valueOf(level).backgroundColor),
         front = {
             FrontCardFace(item.id)
         },
         back = {
             BackCardFace(item, onClickBookmark)
-        }
-    )
+        })
 
     if (cardFace == CardFace.FRONT) {
         LaunchedEffect(key1 = Unit) {
             coroutineScope {
                 launch {
                     animate(
-                        initialValue = 1f,
-                        targetValue = 1.3f,
-                        animationSpec = tween(
-                            durationMillis = 1000,
-                            easing = FastOutSlowInEasing
+                        initialValue = 1f, targetValue = 1.3f, animationSpec = tween(
+                            durationMillis = 1000, easing = FastOutSlowInEasing
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)